ivy/functional/frontends/tensorflow/general_functions.py

```diff
@@ -586,7 +586,12 @@ def strided_slice(
     return ret
 
 
-@with_unsupported_dtypes({"2.14.0 and below": ("uint16",)}, "tensorflow")
+@to_ivy_arrays_and_back
+def tensor_scatter_nd_add(tensor, indices, updates, name=None):
+    return ivy.tensor_scatter_nd_add(tensor, indices, updates)
```

Comment:
It seems ivy functional does not have `tensor_scatter_nd_add` implemented. You can add it or even better if you just use existing functions like `ivy.scatter_nd` and `ivy.add` to implement this